### PR TITLE
Use Hackage packages rather than submodules.

### PR DESCRIPTION
**Pull request by patrickt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -28,13 +28,14 @@ before_install:
 - cabal --version
 
 install:
-- cabal new-update
+- cabal new-update hackage.haskell.org,HEAD
```
Comment (robrix) [inline .travis.yml]: What’s the `HEAD` do?

File: semantic.cabal [hunk 2]
```diff
@@ -304,41 +304,35 @@ library
...
                      , stm-chans ^>= 3.0.0.4
                      , template-haskell ^>= 2.14
                      , time ^>= 1.8.0.2
                      , unliftio-core
                      , unordered-containers ^>= 0.2.9.0
                      , vector ^>= 0.12.0.2
-                     , haskell-tree-sitter
-                     , tree-sitter-go
+                     , tree-sitter-go ^>= 0.1.0.0
                      , tree-sitter-haskell
```
Comment (robrix) [inline semantic.cabal]: No version bound for this?

File: semantic.cabal [hunk 3]
```diff
@@ -304,41 +304,35 @@ library
...
-                     , shelly
+                     , shelly >= 1.5 && <2
                      , split ^>= 0.2.3.3
                      , stm-chans ^>= 3.0.0.4
                      , template-haskell ^>= 2.14
                      , time ^>= 1.8.0.2
                      , unliftio-core
                      , unordered-containers ^>= 0.2.9.0
                      , vector ^>= 0.12.0.2
-                     , haskell-tree-sitter
```
Comment (robrix) [inline semantic.cabal]: Did we drop `tree-sitter` altogether? That’s kind of cool.

File: .travis.yml (hunk 1 above)
Comment (patrickt) [reply .travis.yml]: Made sure we scraped from the absolute latest snapshot. I think we can kill this.

File: semantic.cabal (hunk 2 above)
Comment (patrickt) [reply semantic.cabal]: Whoops. Fixing

Comment (patrickt) [reply semantic.cabal]: Nah, it was just duplicated.

File: semantic.cabal (hunk 3 above)
Comment (patrickt) [reply semantic.cabal]: Nah, it was just duplicated.

**Comment by shapr** [comment]
In my testing, this PR fixes issue #96

**Pull request merged**